lib/utils/export/index.js

```diff
@@ -4,7 +4,9 @@ import { get } from 'lodash';
 import exportNotes from './export-notes';
 import exportToZip from './to-zip';
 
-const exportZipArchive = filename => {
+const filename = 'notes.zip';
```

Comment:
Just for fun, what if we called this `Simplenotes.zip`?

Reply:
Maybe I'm stuffy and overprotective of the Simplenote® brand, but:

> NEVER modify a trademark by changing it to the plural form.

http://www.inta.org/trademarkbasics/factsheets/pages/trademarkusefactsheet.aspx

😂😂😂